docs/guide/animation.md

```diff
@@ -72,7 +72,7 @@ You can specify which easing method to use via the `easing` parameter on the `an
 
 !!! note
 
-    The `textual easing` preview requires the `dev` extras to be installed (using `pip install textual[dev]`).
+    The `textual easing` preview requires the development extras to be installed (using `pip install textual-dev`).
```

Comment:
The `textual` command is not in "extras" any more. Suggest we refer to it as "textual devtools" package. Or explicitly as "textual-dev"